src/api/webhooks.js

```diff
@@ -96,8 +87,6 @@ export class WebhooksApi extends BaseApi {
      */
     delete(webhookId) {
         const url = this.getFullURL('webhooks', webhookId);
-        return this.validateAuthHeaders().then((authHeaders) => {
-            return http('DELETE', url, undefined, authHeaders);
-        });
+        return this.request('DELETE', url);
```

Comment:
@mspensieri Fixed!

@lemieux Yup I tried it with a script and with the web widget and nothing breaks.